ote_cli/ote_cli/utils/io.py

```diff
@@ -62,7 +62,7 @@ def read_model(model_configuration, path, train_dataset):
             "openvino.bin": ModelAdapter(read_binary(path[:-4] + ".bin")),
         }
 
-        for confidence_threshold in ["confidence_threshold", "threshold"]:
+        for confidence_threshold in ["confidence_threshold", "image_threshold"]:
```

Comment:
You can add other metadata keys (image_mean, image_std, pixel_mean, ..) so they can be loaded